### PR TITLE
add compatible terminals guide to doc

### PR DESCRIPTION
**Pull request by Wulingoh**
This pr is to add compatible terminals guide to the doc. 

Notion ticket: https://www.notion.so/centrapay/Add-terminal-compatibility-information-to-docs-site-b0cf326023454c008f4ca446a0118c54

Test plan: 

- [ ] The compatible terminal page should load 
- [ ] The dropdown should load the content once it is click
- [ ] The navigation path is /guides/compatible-terminals#terminal-providers

Preview:
http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com/guides/compatible-terminals/

### CODE REVIEW

File: src/content/guides/compatible-terminals.mdx [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+title: Compatible Terminals
+description: Compatible payment terminals.
+nav:
+  path: Reference/Merchant Integrations
+  order: 1
```
Comment (tom-blake) [inline src/content/guides/compatible-terminals.mdx:6]: @Wulingoh Do you need to bump the order for all the other guides under Merchant Integrations down too?

File: src/content/guides/compatible-terminals.mdx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+import Faq from '../../components/Faq.astro';
+
+The Centrapay service is supported on a range payment and point of sale devices listed below. If the device model is not in this list, it is not supported. If you would like to discuss a new integration or support of Centrapay on a new device, please contact Centrapay at support@centrapay.com
+
+## Terminal Providers
+
+ <Faq>
+  <div>
+    <Disclosure client:visible title="Verifone">
+      Android T650C
```
Comment (tom-blake) [inline src/content/guides/compatible-terminals.mdx:18]: @Wulingoh You can't search "Android T650C" on the site and find this page. What would be involved to make this happen?

File: src/content/guides/compatible-terminals.mdx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  order: 1
+---
+import Disclosure from '../../components/Disclosure.vue';
+import Faq from '../../components/Faq.astro';
+
+The Centrapay service is supported on a range payment and point of sale devices listed below. If the device model is not in this list, it is not supported. If you would like to discuss a new integration or support of Centrapay on a new device, please contact Centrapay at support@centrapay.com
+
+## Terminal Providers
+
+ <Faq>
```
Comment (tom-blake) [inline src/content/guides/compatible-terminals.mdx:15]: @Wulingoh Suggestion: This component is not being used for FAQs - it needs a more generic name

File: src/content/guides/compatible-terminals.mdx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+  <div>
```
Comment (tom-blake) [inline src/content/guides/compatible-terminals.mdx:16]: @Wulingoh Suggestion: Remove all these `divs` if they're not adding anything

File: src/content/guides/compatible-terminals.mdx [hunk 5]
```diff
@@ -0,0 +1,149 @@
+---
+title: Compatible Terminals
+description: Compatible payment terminals.
```
Comment (tom-blake) [inline src/content/guides/compatible-terminals.mdx:3]: @Wulingoh Change: Add more context to this description - it'll be seen in search results and link previews

Something like:
```suggestion
description: A comprehensive list of all of the terminals that Centrapay supports.
```

File: src/content/guides/compatible-terminals.mdx (hunk 3 above)
Comment (Wulingoh) [reply src/content/guides/compatible-terminals.mdx:15]: @tom-blake Unfortunately, this is how the component is name. When I removed the Faq component, it removes the styling. I would have to create another component with similar css.

File: src/content/guides/compatible-terminals.mdx (hunk 4 above)
Comment (Wulingoh) [reply src/content/guides/compatible-terminals.mdx:16]: @tom-blake Similarly, when I removed the div, it also removes the styling. 😢

File: src/content/guides/compatible-terminals.mdx (hunk 1 above)
Comment (Wulingoh) [reply src/content/guides/compatible-terminals.mdx:6]: It is done

File: src/content/guides/compatible-terminals.mdx (hunk 2 above)
Comment (Wulingoh) [reply src/content/guides/compatible-terminals.mdx:18]: @tom-blake The search function is limited to headings only. The dropdown list titles are not headings; they reside within the dropdown itself. Creating headings for these would disrupt the component's structure.

File: src/content/guides/compatible-terminals.mdx (hunk 3 above)
Comment (tom-blake) [reply src/content/guides/compatible-terminals.mdx:15]: @Wulingoh Couldn't you rename the component?

**Review by tom-blake — APPROVED** [review]

File: src/content/guides/compatible-terminals.mdx (hunk 3 above)
Comment (Wulingoh) [reply src/content/guides/compatible-terminals.mdx:15]: @tom-blake Shall I name it as `AccordionListWrapper`

**Pull request merged**